plugins/fuse/server/server_fuse.py

```diff
@@ -422,6 +427,18 @@ def __init__(self, operations, mountpoint, *args, **kwargs):
                 'it empty?  Does the user have permission to create FUSE '
                 'mounts?  It could be another FUSE mount issue, too.' % (
                     mountpoint, ))
+            onError(name)
```

Comment:
It probably makes sense to also pass the Exception to the error handler

Reply:
The Fuse exception is particularly unhandy (just a generic RuntimeError).  While we can pass the exception to the handler, it won't actually be used.

Reply:
I was thinking of API stability, but I suppose it doesn't matter for this plugin. We can add it later if we want.